### PR TITLE
Add open-telekom-cloud

### PR DESCRIPTION
**Pull request by mrIncompetent**

### CODE REVIEW

**Comment by scheeles** [comment]
LGTM from my side
@guusvw @realfake can you the final review

**Review by lbb — APPROVED** [review]
👍

File: api/provider/cloud/otc/provider.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	"github.com/kube-node/nodeset/pkg/nodeset/v1alpha1"
+	"github.com/kubermatic/kubermatic/api"
+	"github.com/kubermatic/kubermatic/api/extensions"
+	"github.com/kubermatic/kubermatic/api/provider"
+	"github.com/kubermatic/kubermatic/api/provider/template"
+	"github.com/kubermatic/kubermatic/api/uuid"
+)
+
+const (
+	tplPath = "/opt/template/nodes/otc.yaml"
```
Comment (guusvw) [inline api/provider/cloud/otc/provider.go:15]: Is this always the same path?

File: api/provider/types.go [hunk 2]
```diff
@@ -182,6 +189,9 @@ func DatacenterCloudProviderName(spec *DatacenterSpec) (string, error) {
 	if spec.Openstack != nil {
 		clouds = append(clouds, OpenstackCloudProvider)
 	}
+	if spec.OTC != nil {
```
Comment (guusvw) [inline api/provider/types.go:192]: Regardless of your PR, should'nt we move this block into an own function?

**Review by guusvw — APPROVED** [review]

**Comment by mrIncompetent** [comment]
We're using now master instead of develop.
Please rebase your branch and open the PR against master

**Pull request closed without merging**